bocoup/threeflows
# Participant: All participants can view and download their data from all scenarios

Pre-test: 

- Log in with a user that has participated in scenario runs. Completeness is not a factor.
- Click "My Scenario Data" in the top navigation menu

Test 1: 

- Click the "Data" icon\* on any Scenario row
- Observe that a data table appeared below the list of scenarios
- Click the "Close" icon\** to close the data table
- Click the "Data" icon\* on the same Scenario row to reopen
- Observe that a data table appeared below the list of scenarios

Test 2: 

- Click the "Data" icon\* on any Scenario row
- Observe that a data table appeared below the list of scenarios
- Click the "Download" icon\*** to download the data
- Observe that a download of the data you are viewing occurs

Test 3: 

- Click the "Scenario title" of any Scenario row
- Observe that you are brought to the "Scenario Run Entry Slide"
- Click the browser's "Back" button, or hit "Backspace"
- Observe that you are brought to the "My Scenario Data" page

Test 4: 

- Click the numbered pagination buttons
- Observe that you can navigate through many lists of scenario runs
- Click the arrow pagination buttons
- Observe that you can navigate through many lists of scenario runs





------------------------------------------------------
\* "Data" icon: 
![image](https://user-images.githubusercontent.com/27985/71013062-0ea83080-20be-11ea-90cb-1f32f859013a.png)

\** "Close" icon: 
![image](https://user-images.githubusercontent.com/27985/71013981-2d5af700-20bf-11ea-83e2-cae346d79495.png)

\*** "Download" icon: 
![image](https://user-images.githubusercontent.com/27985/71013997-36e45f00-20bf-11ea-9491-928d7a63e602.png)
